WordPress/Classes/ViewRelated/Post/PostCell.swift

```diff
@@ -0,0 +1,271 @@
+import UIKit
+import Gridicons
+
+class PostCell: UITableViewCell, ConfigurablePostView {
+    @IBOutlet weak var featuredImageStackView: UIStackView!
+    @IBOutlet weak var featuredImage: CachedAnimatedImageView!
+    @IBOutlet weak var featuredImageHeight: NSLayoutConstraint!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var snippetLabel: UILabel!
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var authorLabel: UILabel!
+    @IBOutlet weak var statusLabel: UILabel!
+    @IBOutlet weak var statusView: UIStackView!
+    @IBOutlet weak var progressView: UIProgressView!
+    @IBOutlet weak var editButton: UIButton!
+    @IBOutlet weak var retryButton: UIButton!
+    @IBOutlet weak var viewButton: UIButton!
+    @IBOutlet weak var moreButton: UIButton!
+    @IBOutlet weak var actionBarView: UIStackView!
+    @IBOutlet weak var containerView: UIView!
+    @IBOutlet weak var upperBorder: UIView!
+    @IBOutlet weak var bottomBorder: UIView!
+    @IBOutlet weak var topPadding: NSLayoutConstraint!
+    @IBOutlet weak var contentStackView: UIStackView!
+    @IBOutlet weak var titleAndSnippetView: UIStackView!
+    @IBOutlet weak var topMargin: NSLayoutConstraint!
+
+    private let margin: CGFloat = WPDeviceIdentification.isiPad() ? 20 : 16
```

Comment:
Let's put all these constants in a constants enum (or two)

Reply:
Done.